lib/workers/repository/update/branch/index.ts

```diff
@@ -117,6 +148,7 @@ export async function processBranch(
       );
       await handlepr(config, existingPr);
       return {
+        configAndManagersHash,
```

Comment:
Not sure we need to return the has if the branch doesn't exist?

Reply:
not necessary upto us.

Reply:
Why add unnecessary complexity?